### PR TITLE
Tests for insights mode

### PR DESCRIPTION
**Pull request by MilanPospisil**
Issue: AAH-2054

Create or adapt existing tests for insights mode.

### CODE REVIEW

File: test/cypress/e2e/insights/login.js [hunk 1]
```diff
@@ -0,0 +1 @@
+/home/milan/software/ansible-hub-ui/test/cypress/e2e/groups_and_users/login.js
```
Comment (himdel) [inline test/cypress/e2e/insights/login.js:1]: relative vs absolute symlinks ;)

Comment (MilanPospisil) [reply test/cypress/e2e/insights/login.js:1]: Ah, make sense, dumb mistake :-)

File: .github/workflows/cypress/cypress.env.json.insights [hunk 2]
```diff
@@ -5,5 +5,7 @@
   "username": "admin",
   "password": "admin",
   "galaxykit": "galaxykit --ignore-certs --auth-url 'http://localhost:8002/auth/realms/redhat-external/protocol/openid-connect/token'",
-  "insightsLogin": true
+  "insightsLogin": true,
+  "disableRepoSwitch" : true,
+  "namespaceName" : "Partners"
```
Comment (himdel) [inline .github/workflows/cypress/cypress.env.json.insights:10]: what is this trying to do?

I only see it used as an url fragment in goToNamespaces .. but we definitely don't have any routes with capital letters

If you're already relying on insights mode, why not just use the whole route?
Or, if you want this clean, drop the logic based on mode and have a setting for the namespaces url to go to

File: test/cypress/e2e/namespaces/namespace_detail.js [hunk 3]
```diff
@@ -77,28 +77,30 @@ describe('Namespace detail screen', () => {
     // The test for success are impmeneted in the collection_upload file
   });
 
-  it('should filter by repositories', () => {
-    const namespace = 'coolestnamespace';
-    cy.galaxykit('-i namespace create', namespace);
-    cy.visit(`${uiPrefix}repo/published/${namespace}`);
+  if (!disableRepoSwitch) {
```
Comment (himdel) [inline test/cypress/e2e/namespaces/namespace_detail.js:80]: would it make more sense to move repo switcher tests to a separate file instead?

(They will go away once repository management is in, so .. either works.)

File: .github/workflows/cypress/cypress.env.json.insights [hunk 4]
```diff
@@ -5,5 +5,6 @@
   "username": "admin",
   "password": "admin",
   "galaxykit": "galaxykit --ignore-certs --auth-url 'http://localhost:8002/auth/realms/redhat-external/protocol/openid-connect/token'",
-  "insightsLogin": true
+  "insightsLogin": true,
+  "disableRepoSwitch" : true,
```
Comment (himdel) [inline .github/workflows/cypress/cypress.env.json.insights:9]: trailing commas are not valid json

File: test/cypress/e2e/namespaces/namespace_detail.js (hunk 3 above)
Comment (MilanPospisil) [reply test/cypress/e2e/namespaces/namespace_detail.js:80]: So if they will go away soon, why just not leave them here for now?

**Review by himdel — APPROVED** [review]

**Pull request merged**